tracing-subscriber/src/layer.rs

```diff
@@ -994,6 +994,29 @@ where
         }
     }
 
+    /// Finds the [`SpanRef`] associated with an [`Event`], if it has one.
+    ///
+    /// Compared to [`Context::current_span`] and [`Context::lookup_current`],
+    /// this respects overrides provided by the [`Event`].
+    ///
+    /// Compared to [`Event::parent`], this automatically falls back to the contextual
+    /// span, if required.
```

Comment:
it would be nice to have an example here, but it's okay if you don't have the time to add one.

Reply:
the docs for other `Context` methods that require the `LookupSpan` trait have notes about how a `Layer` needs to add a `LookupSpan` bound to use those methods: https://github.com/tokio-rs/tracing/blob/01415a7b4046406a7321a1564c88db6d5e24fa0b/tracing-subscriber/src/layer.rs#L1089-L1098

Can we add that here?
```suggestion
    /// span, if required.
    ///
    /// <div class="information">
    ///     <div class="tooltip ignore" style="">ⓘ<span class="tooltiptext">Note</span></div>
    /// </div>
    /// <div class="example-wrap" style="display:inline-block">
    /// <pre class="ignore" style="white-space:normal;font:inherit;">
    /// <strong>Note</strong>: This requires the wrapped subscriber to implement the
    /// <a href="../registry/trait.LookupSpan.html"><code>LookupSpan</code></a> trait.
    /// See the documentation on <a href="./struct.Context.html"><code>Context</code>'s
    /// declaration</a> for details.
    /// </pre></div>
```